### PR TITLE
Milestone2/basic features

### PR DESCRIPTION
**Pull request by YoseptF**
In this milestone I created 3 mandatory components:
- Display: To show the current state of the calculator
- Button: To let the user interact with the clasculator
- buttonPanel: to hold all the buttons inside

image

I also added a couple more components to make it look prettier:
- titlebar
- menu
- window

### CODE REVIEW

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <div className="group2">
+      <Button name="7" />
+      <Button name="8" />
+      <Button name="9" />
+      <Button name="X" />
+    </div>
+    <div className="group3">
+      <Button name="4" />
+      <Button name="5" />
+      <Button name="6" />
```
Comment (bellom) [inline src/components/ButtonPanel.js]: ### [Optional]
I think we could make use of two loops to make the Button instead of repeating ourselves. let's practice [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code.

To do this; create an array of symbols and loop through the array using [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.

File: src/components/Display.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      value={Number(value).toString()}
+      onChange={handleChange}
+    />
+  );
+};
+
+
+Display.propTypes = {
+  value: PropTypes.number.isRequired,
+  onChange: PropTypes.func.isRequired,
```
Comment (bellom) [inline src/components/Display.js]: **You did a good job adding a few more components.**
_Note: for every new component or a new way of building the project, it as to go with the requirements specified by Microverse._
To avoid future code review requests, I will advise you to stick to Microverse requirements.
<hr>
Always read the pathwright instructions before starting the project to avoid the need of sending another code review, and always try as much as possible to stick to the requirements.

**Issues:**


_1. seems you're having `value` as `result`

_2. `result` should be a string_

_3. the `result` default value should be set to 0._

Screenshot from pathwright: 
image

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @YoseptF :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly update your `README` file to look more professional**.
Checkout out the below link for help, free free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/


**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by YoseptF** [comment]
Hi @bellom,
Thank you for the response, I already updated both mandatory and optional changes.
I wanted about the README file because I checked the suggested template and it seems to be very similar to what I have.
We both have:
- Title
- Built with
- Live demo
- Development Setup
- Authors

We differ with:
- Header Image (I still don't have a screenshot of the final product)
- Contributing section (I don't find it useful in this project)
- License (I don't have a license for this project)

---

In my first milestone, I didn't add a README file since I don't have the whole project to make a proper screenshot, but I was asked to do so and then update it later down the protect, which I took to mean `add screenshot later`, but now I'm thinking maybe I'm forgetting to add something important in my readme file since it's the second time I'm asked to modify the Readme.

---

TL;DR -> Am I missing something in my readme file? I updated it to look even more similar than before, but I'm worried I'm missing something important in there.

As always thanks for your time, and happy coding.

**Review by bellom — APPROVED** [review]
## Hi @YoseptF  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**
